elements/pl-file-preview/pl-file-preview.mustache

```diff
@@ -16,6 +16,11 @@
               <p class="file-status">uploaded</p>
             </div>
             <div class="file-status-container-right">
+              <a download="{{name}}" class="btn btn-outline-secondary btn-sm"
+                 onclick="event.stopPropagation();"
```

Comment:
Is this `onclick` meant to prevent redirects? Seems to have no effect on my end whether it's there or not (firefox, chrome, edge).

Reply:
It is to prevent the preview to open or close when download is pressed.

Reply:
Ah nice 👍